.github/workflows/test.yml

```diff
@@ -99,7 +99,7 @@ jobs:
           ls
           mv wasm build/wasm
           yarn install --frozen-lockfile
-          docker run -d --rm -p 9944:9944 kiltprotocol/portablegabi-node --dev --ws-port 9944 --ws-external
```

Comment:
These sub commands are [used by default](https://github.com/KILTprotocol/portablegabi-node/blob/wf-568-fix-docker/Dockerfile#L84) when running the docker file.